tests/AutoReview/FixerFactoryTest.php

```diff
@@ -132,6 +132,8 @@ public function provideFixersPriorityCases()
             [$fixers['ordered_class_elements'], $fixers['no_blank_lines_after_class_opening']], // tested also in: ordered_class_elements,no_blank_lines_after_class_opening.test
             [$fixers['ordered_class_elements'], $fixers['space_after_semicolon']], // tested also in: ordered_class_elements,space_after_semicolon.test
             [$fixers['php_unit_fqcn_annotation'], $fixers['no_unused_imports']], // tested also in: php_unit_fqcn_annotation,unused_use.test
+            [$fixers['php_unit_no_expectation_annotation'], $fixers['no_empty_phpdoc']], // tested also in: php_unit_no_expectation_annotation,no_empty_phpdoc.test
+            [$fixers['php_unit_no_expectation_annotation'], $fixers['php_unit_expectation']], // tested also in: php_unit_no_expectation_annotation,php_unit_expectation.test
```

Comment:
all new fixers dealing with `PHPDoc`'s should go after `phpdoc_to_comment`

Reply:
> all new fixers dealing with `PHPDoc`'s should go after `phpdoc_to_comment`

blindly? I disagree
`phpdoc_to_comment` changes phpdocs for non-structural elements.
`php_unit_no_expectation_annotation` works on it's own, it checks if given docblock is for real function (so it filter out phpdocs that would be changed by `phpdoc_to_comment` to regular comment)


> and after `general_phpdoc_annotation_remove`

actually, I would put it before, as I expect annotation to be changed into code than dropped when having both fixers configured.
*BUT*, `general_*` fixers are tricky and kind of "advanced" usage. we don't cover priority nor conflicts for them at all. as they could be configured to do very different things.
regular fixer is changing sth semantically, `general_*` is changing sth lexical(-ly?), without knowing the context.

Reply:
agreed; as in no changes needed here :+1: 

I give the `general_*` case a bit more thought, it _might_ make sense to put those on a very high or low priority so no matter what we decide it will work in consistent way